### ISSUE TITLE
first run issue

### ISSUE THREAD

**Issue by LukeOeding** [jcjohnson/neural-style#66]
Hi! I'm trying to install neural-style on my 2015 iMac, but I'm having some issues. I followed the installation guide, and got to the final step to check the installation, but I had an error, but I'm not sure what to do to fix it. Can anyone help? Thanks!

    oeding@Lukes-iMac:~/Image Processing/neural-style-master$ th neural_style.lua -gpu -1 -print_iter -1
    [libprotobuf WARNING google/protobuf/io/coded_stream.cc:537] Reading dangerously large protocol message.  If the message turns out to be larger than 1073741824 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
    [libprotobuf WARNING google/protobuf/io/coded_stream.cc:78] The total number of bytes read was 574671192
    Successfully loaded models/VGG_ILSVRC_19_layers.caffemodel
    /Users/oeding/torch/install/bin/luajit: models/VGG_ILSVRC_19_layers_deploy.prototxt.cpu.lua:2: bad argument #1 to 'insert' (table expected, got nil)
    stack traceback:
	[C]: in function 'insert'
	models/VGG_ILSVRC_19_layers_deploy.prototxt.cpu.lua:2: in main chunk
	[C]: in function 'dofile'
	./loadcaffe_wrapper.lua:48: in function 'load'
	neural_style.lua:65: in function 'main'
	neural_style.lua:454: in main chunk
	[C]: in function 'dofile'
	...ding/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:133: in main chunk
	[C]: at 0x01082f32d0

**Comment by dodekeract** [comment]
Most likely related to and solved by #62.

**Comment by jcjohnson** [comment]
Looks like the same problem as https://github.com/jcjohnson/neural-style/issues/62.

Update neural-style and luarocks to the latest versions.

**Comment by LukeOeding** [comment]
Hi Justin,

Thanks for the pointer. That fixed the problem!


One thing I thought I would pass along: I'm using a mac, so I thought I would point out that in the install doc it tells me to 
sudo apt-get install libprotobuf-dev protobuf-compiler
I had to do something else since apt-get is not native to my machine.  I think ultimately I figured out how to do the same thing with HomeBrew, but perhaps it would be good to have something to that effect in the install doc for future users.

Cheers,

Luke
